pyannote/database/loader.py

```diff
@@ -187,6 +187,55 @@ def __call__(self, file: ProtocolFile) -> Timeline:
         return self.loaded_[uri]
 
 
+class LABLoader:
+    """LAB loader
+
+    Parameters
+    ----------
+    path : str
+        Path to LAB file with mandatory {uri} placeholder.
+        (e.g. "/path/to/{uri}.lab")
+
+        each .lab file contains the segments for a single audio file, in the following format:
+        start end label
+
+        ex.
+        0.0 12.3456 sing
+        12.3456 15.0 nosing
+        ...
+    """
+
+    def __init__(self, path: Text = None):
+        super().__init__()
+
+        self.path = str(path)
+        
+        _, placeholders, _, _ = zip(*string.Formatter().parse(self.path))
+        self.placeholders_ = set(placeholders) - set([None])
```

Comment:
Since `LAB` format does not have a `uri` (it only is `start end label`), the `{uri}` placeholder in `path` is mandatory.

Therefore, we should probably raise an error here if it does not. Something like:

```suggestion
        self.placeholders_ = set(placeholders) - set([None])
        if "uri" not in self.placeholders_:
            raise ValueError("`path` must contain the {uri} placeholder.")
```